src/jit/codegenarm64.cpp

```diff
@@ -862,18 +874,90 @@ void                CodeGen::genFuncletProlog(BasicBlock* block)
     if (isFilter)
     {
         // This is the first block of a filter
+        // Note that register x1 = CallerSP of the containing function
+        // X1 is overwritten by the first Load (new callerSP)
+        // X2 is scratch when we have a large constant offset
```

Comment:
This is a bit of an aside for this PR, but why is regTracker being used here? It shouldn't be used for the RyuJIT backend (though I think there are some remaining uses in code that was simply brought over as-is). Also, is X2/REG_R2 specified by some convention, or should this be allocated as a temp reg during register allocation?